### PR TITLE
Compress HTML

### PR DESCRIPTION
**Pull request by Androbin**
Towards #130
#243 @sushain97 /cc

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -134,7 +134,9 @@ build/l10n-rel.html: assets/strings/locales.json isobork build/.d
 	awk 'BEGIN{while(getline<"isobork")i[$$1]=$$2} /:/{sub(/^[^"]*"/,""); sub(/".*/,""); borkd=i[$$0]; if(!borkd)borkd=$$0; print "<link rel=\"alternate\" hreflang=\""borkd"\" href=\"index."$$0".html\">"}' $^ > $@
 
 build/index.%.html: build/strings/%.json build/index.localiseme.html $(CONFIG) tools/read-conf.py tools/localise-html.py
-	./tools/localise-html.py -c $(CONFIG) build/index.localiseme.html $< $@
+	./tools/localise-html.py -c $(CONFIG) build/index.localiseme.html $< $@.tmp
```
Comment (sushain97) [inline Makefile]: Seems possible to just pipe into `htmlmin`.

Comment (Androbin) [reply Makefile]: The script expects an output path. It doesn't print to `stdout`.

Comment (sushain97) [reply Makefile]: https://htmlmin.readthedocs.io/en/latest/quickstart.html#command-line

Comment (Androbin) [reply Makefile]: I was referring to `localise-html.py` not printing to `stdout`.
I am well aware that I could pipe that into `htmlmin` if it did.

Comment (sushain97) [reply Makefile]: Change it then?

**Comment by Androbin** [comment]
* `tools/localise-html.py` now prints to `stdout`
* `Makefile` adapted accordingly

**Comment by sushain97** [comment]
How will this work if `htmlmin` isn't available?

**Comment by Androbin** [comment]
I can provide a fallback like last time. But then we'd better return to writing the input html to a file first.

**Comment by Androbin** [comment]
Should work now.

**Comment by Androbin** [comment]
I was not quite sure about the other HTML targets.
Are these just intermediate build steps or do we need to do this for other files as well?

**Comment by Androbin** [comment]
@sushain97 Anything else to consider?

**Pull request merged**